src/model/row.py

```diff
@@ -0,0 +1,70 @@
+import uuid
+import os
+import json
+import pathlib
+
+class Row:
+    def __init__(self, table, data):
+        self.table = table
+        self.data = data
+        self.primary_key = self.__get_primary_key__(data.get(table.get_primary_key()))
```

Comment:
You don’t need to pass a variable to get_primary_key.
It should be void

Reply:
self.data[table.get_primsy_key] = __get_prinay_key_internal()

get_primary_key () // rerun data [primary_key]